contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -714,6 +714,128 @@ $BODY$
 STRICT
 LANGUAGE SQL IMMUTABLE;
 
+CREATE OR REPLACE FUNCTION sys.DATETIMEOFFSETFROMPARTS(IN p_year NUMERIC,
+                                                               IN p_month NUMERIC,
+                                                               IN p_day NUMERIC,
+                                                               IN p_hour NUMERIC,
+                                                               IN p_minute NUMERIC,
+                                                               IN p_seconds NUMERIC,
+                                                               IN p_fractions NUMERIC,
+                                                               IN p_hour_offset NUMERIC,
+                                                               IN p_minute_offset NUMERIC,
+                                                               IN p_precision NUMERIC)
+RETURNS sys.DATETIMEOFFSET
+AS
+$BODY$
+DECLARE
+    v_err_message VARCHAR;
+    v_fractions VARCHAR;
+    v_precision SMALLINT;
+    v_calc_seconds NUMERIC; 
+    v_resdatetime TIMESTAMP WITHOUT TIME ZONE;
+    v_string TEXT;
+    v_sign TEXT;
```

Comment:
nit; it would be nice to state `search_path` explicitly, e.g., `pg_catalog.text`, `sys.varchar`, `pg_catalog.varchar`, etc..